pan-domain-auth-core/src/main/scala/com/gu/pandomainauth/PanDomainAuthSettingsRefresher.scala

```diff
@@ -32,45 +36,34 @@ class PanDomainAuthSettingsRefresher(
   awsRegion: Option[Region] = Option(Region getRegion Regions.EU_WEST_1),
   proxyConfiguration: Option[ProxyConfiguration] = None
 ) {
+  private implicit val ec: ExecutionContext = actorSystem.dispatcher
   lazy val bucket = new S3Bucket(awsCredentialsProvider, awsRegion, proxyConfiguration)
 
-  private lazy val settingsMap = bucket.readDomainSettings(domain)
-  private lazy val authSettings: Agent[PanDomainAuthSettings] = Agent(PanDomainAuthSettings(settingsMap))
+  private lazy val settingsMap = bucket.readDomainSettingsBlocking(domain)
```

Comment:
Blocking during setup as we can't proceed without the initial value of the settings